osquery/dispatcher/scheduler.cpp

```diff
@@ -171,6 +172,8 @@ void SchedulerRunner::start() {
   // Start the counter at the second.
   auto i = osquery::getUnixTime();
   for (; (timeout_ == 0) || (i <= timeout_); ++i) {
+    monitoring::record("dispatcher.scheduler_runner.step_time",
+                       static_cast<monitoring::ValueType>(i));
```

Comment:
lets do not put code for actual logging in this PR, because we need to have completely different discussion on what to log, what do you think?

Reply:
It was just an example. Ok, I'll remove it.